### PR TITLE
Updates for research group Wick

### PR DESCRIPTION
**Pull request by jpthiele**
These are new or missing publications of the research group of @tommeswick .

### CODE REVIEW

**Review by peterrum — COMMENTED** [review]
Very nice! Would you mind to make the bibtex entries a bit more consistent?

**Comment by jpthiele** [comment]
Now all additions are consistent and all checks pass. 
I was not sure whether I should make all the auto assigned bibtex identifiers consistent as well.
If yes, then I'll happily do that too.

**Review by peterrum — APPROVED** [review]
Good to go once you have resolved the merge conflicts.

File: publications-2021.bib
Comment (marcfehling) [inline publications-2021.bib:672]: Is it still a preprint? I found this article here: https://link.springer.com/chapter/10.1007/978-3-030-87312-7_24

Could you add the doi and update the journal etc.?

File: publications-2021.bib
Comment (marcfehling) [inline publications-2021.bib:687]:
```suggestion
  doi = {10.1002/pamm.202100151},
```

File: publications-2021.bib
Comment (marcfehling) [inline publications-2021.bib:701]:
```suggestion
  doi = {10.1002/pamm.202100175},
```

File: publications-2021.bib
Comment (marcfehling) [inline publications-2021.bib:715]:
```suggestion
  doi = {10.1002/pamm.202100174},
```

File: publications-2021.bib
Comment (marcfehling) [inline publications-2021.bib:724]:
```suggestion
  title={Efficient monolithic solvers for fluid-structure interaction applied to flapping membranes},
```

File: publications-2021.bib
Comment (marcfehling) [inline publications-2021.bib:734]:
```suggestion
  title={Parallel domain decomposition solvers for the time harmonic Maxwell equations},
```

File: publications-2021.bib
Comment (marcfehling) [inline publications-2021.bib:744]:
```suggestion
  title={Adjoint-based methods for optimization and goal-oriented error control applied to fluid-structure interaction: implementation of a partition-of-unity dual-weighted residual estimator for stationary forward FSI problems in deal.II},
```

File: publications-2021.bib
Comment (marcfehling) [inline publications-2021.bib:747]:
```suggestion
  url = {10.4995/YIC2021.2021.12332},
```

File: publications-2021.bib
Comment (marcfehling) [inline publications-2021.bib:760]:
```suggestion
  doi = {10.1016/j.engfracmech.2021.108060},
```

File: publications-2022.bib
Comment (marcfehling) [inline publications-2022.bib:122]:
```suggestion
  doi = {10.1016/j.cam.2022.114315},
```

File: publications-2022.bib
Comment (marcfehling) [inline publications-2022.bib:105]: Can you update the entry according to https://link.springer.com/article/10.1007/s00466-022-02147-0

File: publications-2022.bib
Comment (marcfehling) [inline publications-2022.bib:88]:
```suggestion
  title={Space-time error control using a partition-of-unity dual-weighted residual method applied to low mach number combustion},
```

File: publications-2020.bib
Comment (marcfehling) [inline publications-2020.bib:1026]: Not mandatory, but an URL would be nice if one exists. Can this thesis be found anywhere online?

**Review by marcfehling — CHANGES_REQUESTED** [review]

File: publications-2020.bib
Comment (jpthiele) [reply publications-2020.bib:1026]: Sadly, no.

File: publications-2021.bib
Comment (jpthiele) [reply publications-2021.bib]: Yes indeed, that is already published by now. I updated the entry

File: publications-2021.bib
Comment (marcfehling) [inline publications-2021.bib:754]:
```suggestion
  doi = {10.4995/YIC2021.2021.12332},
```

**Review by marcfehling — APPROVED** [review]
Skimming through the PR again, I found one small remaining issue.

It will be ready for merge, once you rebased on the master branch. It would be great if you squashed the commits into one in the process.

**Comment by marcfehling** [comment]
I took the liberty and made the according changes directly on your branch.

**Pull request merged**
